docs/getting-started/server/events.md

```diff
@@ -128,17 +161,15 @@ end
         "password": "SET_A_PASSWORD_HERE_123",
         "exchangeName": "events-exchange",
         "eventRepositoryQueueName": "events-write-queue",
+        "httpPostQueueName": "events-httpPost-queue",
       }
+      "httpPostUrl": "<HTTP POST URL>",
     }
     ```
 
-2.  (optional) To use the RabbitMqEventHttpPostListener, specify a queue name and destination URL
-    that will receive the POST inside the RabbitMq object:
-
-    ```json
-        "httpPostQueueName": "events-httpPost-queue",
-        "httpPostUrl": "<HTTP POST URL>",
-    ```
+2.  (optional) The `httpPostQueueName` and `httpPostUrl` specified above are optional. If they are
+    defined, an `HttpPostEventHandler` will be added to a `RabbitMqEventListenerService` instance
+    that will `POST` the event to the configured URL.
 
     - Tip: [RequestBin](http://requestbin.com/) provides an easy to set up server that will receive
```

Comment:
🎨 Missed this but there should be some kind of info box we can use here instead of "Tip:".